### PR TITLE
pcrm 5555, pcrm 6602 mesh test and fixes

### PR DESCRIPTION
**Pull request by ranisen**
Small fixes for MESH required to pass the Solutions Assurance testing.
Also a small bug fix for PCRM-6602

### CODE REVIEW

File: mesh_aws_client/mesh_fetch_message_chunk_application.py [hunk 1]
```diff
@@ -253,13 +252,15 @@ def start(self):
 
         parts_read = 0
         part_buffer = b""
+        bytes_read = 0
 
         is_report = self.http_response.headers.get("Mex-Messagetype") == "REPORT"
         if is_report:
             buffer = json.dumps(dict(self.http_response.headers))
             etag = self._upload_part_to_s3(buffer, True, True)
```
Comment (Hobdell) [inline mesh_aws_client/mesh_fetch_message_chunk_application.py]: etag variable is not used after assigning here

File: mesh_aws_client/mesh_send_message_chunk_application.py [hunk 2]
```diff
@@ -110,7 +110,6 @@ def start(self):
 
         is_finished = self.input.get("complete", False)
         if is_finished:
-            # TODO log error
             self.response.update({"statusCode": HTTPStatus.INTERNAL_SERVER_ERROR.value})
             raise SystemError("Already completed upload to MESH")
```
Comment (Hobdell) [inline mesh_aws_client/mesh_send_message_chunk_application.py:114]: Critical/ Error log reference missing

File: mesh_aws_client/mesh_fetch_message_chunk_application.py (hunk 1 above)
Comment (ranisen) [reply mesh_aws_client/mesh_fetch_message_chunk_application.py]: True

File: mesh_aws_client/mesh_send_message_chunk_application.py (hunk 2 above)
Comment (ranisen) [reply mesh_aws_client/mesh_send_message_chunk_application.py:114]: Raising 'SystemError' will create a critical log from the logging infrastructure.

**Comment by ranisen** [comment]
Reviewed by Kat Hobdell - review fixes applied, and package needs building today but she's off this afternoon!

**Pull request merged**